content/livesync/quickstart.textile

```diff
@@ -1,7 +1,240 @@
 ---
 title: Quickstart
-meta_description: "."
+meta_description: "A quickstart guide to learn the basics of integrating the Ably LiveSync product into your application."
 product: livesync
 languages:
   - javascript
 ---
+
+This quickstart will provide you with a set of instructions to get an example running locally on your machine. Following this, you'll cover each component of LiveSync found within the code of the example.
+
+h2(#example). The example
+
+In this section you will cover all the steps required to get an existing example running locally on your machine.
+
+h3(#step-0). Clone the repository
+
+Clone the "live comments repository":https://github.com/ably-labs/live-comments:
+
+```[sh]
+git clone git@github.com:ably-labs/live-comments.git
```

Comment:
Why `git@github.com:`?

Reply:
No reason, it can be done either way. But I've updated to HTTPS instead.